tests/test_enterprise/api/test_views.py

```diff
@@ -8809,6 +8809,31 @@ def test_assign_learners_revives_previously_removed_members(self):
         assert pending_membership.status == GROUP_MEMBERSHIP_PENDING_STATUS
         assert membership.status == GROUP_MEMBERSHIP_ACCEPTED_STATUS
 
+    # TODO: Figure out how to make sqlite mirror MySQL case-insensitivity for IN queries
```

Comment:
The following line does a case insensitive lookup under MySQL, but does not under the database engine our unit tests use, causing this unit test to fail.  
https://github.com/openedx/edx-enterprise/blob/db8ac49972b490a8dd0d88c24d9576cba4a54379/enterprise/api/v1/views/enterprise_group.py#L313
While alternative lookup schemes exist, the ones I've found that are database-independent would incur a performance hit of some kind.